code/datums/mind.dm

```diff
@@ -201,6 +201,8 @@ GLOBAL_LIST_EMPTY(personal_objective_minds)
 						referred_gender = "Androgynous"
 				M.known_people[H.real_name]["FGENDER"] = referred_gender
 				M.known_people[H.real_name]["FAGE"] = H.age
+				if(H.dna && H.dna.species)
+					known_people[H.real_name]["FSPECIES"] = H.dna.species.name
```

Comment:
this is runtiming with bad index

<img width="869" height="391" alt="Image" src="https://github.com/user-attachments/assets/900f88cd-9500-4d93-87cb-545d94d7028c" />